src/components/Notifications/NotificationGroupCard/NotificationGroupCard.tsx

```diff
@@ -0,0 +1,115 @@
+    disabled,
+    subscriptionDetails,
+    error,
+    onChangeNotificationSetting,
+    onClearChangeNotificationSettingError
+  } = props
+
+  const [showError, setShowError] = useState(false)
+
+  useEffect(() => {
+    if (error) {
+      setShowError(true)
+    }
+  }, [error])
+
+  const handleOnChangeNotificationSetting = useCallback(
+    (checked: boolean, type: NotificationType) => {
+      if (!isLoading) {
+        const subscriptionDetailsChanged = {
+          ...subscriptionDetails,
+          messageType: { ...subscriptionDetails.messageType, [toCamel(type)]: { inApp: true, email: checked } }
+        }
+        onChangeNotificationSetting(objectToSnake(subscriptionDetailsChanged))
+      }
+    },
+    [subscriptionDetails, onChangeNotificationSetting, isLoading]
+  )
+
+  const handleClose = useCallback(() => {
+    setShowError(false)
+    onClearChangeNotificationSettingError()
+  }, [])
+
+  return (
+    <>
+      <AccordionStyled defaultExpanded={defaultExpanded}>
+        <AccordionSummaryStyled expandIcon={<ExpandMoreIcon />}>
+          <AccordionSummaryContainerStyled>
+            {isLoading ? (
+              <Skeleton animation="wave" width={100} height={20} data-testid={NOTIFICATION_CARD_LOADING_TEST_ID} />
+            ) : (
+              <AccordionTitleStyled data-testid={NOTIFICATION_CARD_TITLE_TEST_ID}>
+                {t(`settings.notifications.subscription_group_label_${subscriptionGroupKeys}`)}
+              </AccordionTitleStyled>
+            )}
+            {isLoading ? (
+              <Skeleton animation="wave" width={200} height={24} />
+            ) : (
+              <AccordionDescriptionStyled data-testid={NOTIFICATION_CARD_DESCRIPTION_TEST_ID}>
+                {t(`settings.notifications.subscription_group_description_${subscriptionGroupKeys}`)}
+              </AccordionDescriptionStyled>
+            )}
+          </AccordionSummaryContainerStyled>
+        </AccordionSummaryStyled>
+        <AccordionDetailsStyled>
+          {notificationTypesInGroup.map(type => (
+            <NotificationItemContainerStyled key={type}>
+              <NotificationItemTextIconContainerStyled>
+                <NotificationIcon name={type} />
+                <NotificationItemTextStyled>{t(`settings.notifications.types.${type}`)}</NotificationItemTextStyled>
+              </NotificationItemTextIconContainerStyled>
+              {!disabled && (
+                <Switch
+                  onChange={(_, checked) => handleOnChangeNotificationSetting(checked, type)}
+                  checked={subscriptionDetails.messageType[toCamel(type)].email}
+                  disabled={isSavingSubscription}
+                  data-testid={NOTIFICATION_CARD_SWITCH_TEST_ID}
+                />
+              )}
+            </NotificationItemContainerStyled>
+          ))}
+        </AccordionDetailsStyled>
+      </AccordionStyled>
+      <Snackbar open={showError} onClose={handleClose} autoHideDuration={5000}>
```

Comment:
Would you mind adding a test for the show error toast if possible? We should check if it's rendered and what happens when closed